app/build.gradle

```diff
@@ -8,14 +8,14 @@ repositories {
 
 android {
     compileSdkVersion 24
-    buildToolsVersion '24.0.2'
+    buildToolsVersion '25.0.0'
 
     defaultConfig {
         applicationId "xyz.jhughes.laundry"
         minSdkVersion 16
         targetSdkVersion 24
-        versionCode 9
-        versionName "2.3"
+        versionCode 10
```

Comment:
Wait. Why is this change on your branch? Are you preemptively upticking it for a release?